pkg/util/testutil/conditions.go

```diff
@@ -1,3 +1,4 @@
+// TODO: move away from pkg, this code is only usable from tests
```

Comment:
do we have a ticket for this?

Reply:
Yes [CLOUDP-221195](https://jira.mongodb.org/browse/CLOUDP-221195)